locales/de-de.json

```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "Was du bist? (Wähle jetzt aus, du kannst es später ändern)",
```

Comment:
> Was du bist?

-> Was bist du?

I would also suggest changing the text in parenthesis to

(Deine Auswahl kann später geändert werden)